reference/halyard/custom.md

```diff
@@ -79,6 +79,7 @@ cluster.
 | `enabled` | Whether or not this service is enabled. Be careful, since disabling critical services like `consul-client` can make your entire installation unusable and un-upgradable. |
 | `env` | Environment variables to make available to this Service. Supplied as YAML key-value pairs. |
 | `healthEndpoint` | The HTTP endpoint that can be queried for health status.  Leave empty if you want Halyard to simply check that a TCP connection is openable. This information is typically transmitted to the platform's discovery system. |
+| `useExecHealthCheck` | Determines whether the kubernetes liveness check should use `exec` or `httpGet` method (`exec` is required for some istio compantibilty and is default, `httpGet` is a bit more "pure" and is required for some Ingress Controllers). | 
```

Comment:
@paulczar Can you move this entry to the end of the table?  Because alphabet. 
:grin: 

Also, where you have "...is default, `httpGet` is...
s/,/./

Also, would you be able to be a little more specific about what "pure" means here, if you can do so without blowing up the size of this entry.

Thanks!